src/api/notify-namespace.ts

```diff
@@ -0,0 +1,529 @@
+import { AlchemyConfig } from './alchemy-config';
+import {
+  AddressActivityResponse,
+  AddressActivityWebhook,
+  AddressWebhookParams,
+  AddressWebhookUpdate,
+  DroppedTransactionWebhook,
+  GetAddressesOptions,
+  GetAllWebhooksResponse,
+  MinedTransactionWebhook,
+  Network,
+  NftActivityWebhook,
+  NftFilter,
+  NftFiltersResponse,
+  NftWebhookParams,
+  NftWebhookUpdate,
+  TransactionWebhookParams,
+  Webhook,
+  WebhookType,
+  WebhookVersion
+} from '../types/types';
+import { requestHttpWithBackoff } from '../internal/dispatch';
+import { AlchemyApiType } from '../util/const';
+import {
+  RawAddressActivityResponse,
+  RawCreateWebhookResponse,
+  RawGetAllWebhooksResponse,
+  RawNftFilterParam,
+  RawNftFiltersResponse,
+  RawWebHook
+} from '../internal/raw-interfaces';
+import { AxiosRequestConfig, Method } from 'axios';
+
+/**
+ * The Notify namespace contains methods used for creating, reading, updating,
+ * and deleting webhooks in the Notify API.
+ *
+ * To use the methods in the API, you must provide your team's auth token in the
+ * {@link AlchemySettings.notifyAuthToken} field when configuring
+ * {@link AlchemySettings}. The auth token can be found in the Alchemy Dashboard
+ * on the Notify tab.
+ *
+ * Note that not all networks are supported in the Notify API. Please consult
+ * the documentation for which networks are supported.
+ *
+ * Do not call this constructor directly. Instead, instantiate an Alchemy object
+ * with `const alchemy = new Alchemy(config)` and then access the notify
+ * namespace via `alchemy.notify`.
+ */
+export class NotifyNamespace {
+  constructor(private readonly config: AlchemyConfig) {}
+
+  /** Get all webhooks from every app on your team. */
```

Comment:
Nit: Since webhooks are not always tied with apps we might want to rename to `Get all webhooks on your team.`